MapboxNavigation/FeedbackItem.swift

```diff
@@ -13,18 +13,74 @@ public extension FeedbackType {
     // TODO: Localize these strings
     internal var title: String {
         switch self {
-            case .general:
-                return "Feedback"
-            case .incorrectVisual(_):
-                return "Incorrect Visual"
-            case .confusingAudio(_):
-                return "Confusing Audio"
-            case .routeQuality(_):
-                return "Route Quality"
-            case .illegalRoute(_):
-                return "Illegal Route"
-            case .roadClosure(_):
-                return "Road Closure"
+        case .general:
+            return "Feedback"
+        case .incorrectVisual(.none):
+            return "Incorrect Visual"
+        case .incorrectVisual(.turnIconIncorrect):
+            return "Turn Icon Incorrect"
+        case .incorrectVisual(.streetNameIncorrect):
+            return "Street Name Incorrect"
+        case .incorrectVisual(.instructionUnnecessary):
+            return "Instruction Unnecessary"
+        case .incorrectVisual(.instructionMissing):
+            return "Instruction Missing"
+        case .incorrectVisual(.maneuverIncorrect):
+            return "Maneuver Incorrect"
+        case .incorrectVisual(.exitInfoIncorrect):
+            return "Exit Info Incorrect"
+        case .incorrectVisual(.laneGuidanceIncorrect):
+            return "Lane Guidance Incorrect"
+        case .incorrectVisual(.roadKnownByDifferentName):
+            return "Road Known By Different Name"
+        case .incorrectVisual(.other):
+            return "Other"
+        case .confusingAudio(.none):
+            return "Confusing Audio"
+        case .confusingAudio(.guidanceTooEarly):
+            return "Guidance Too Early"
+        case .confusingAudio(.guidanceTooLate):
+            return "Guidance Too Late"
+        case .confusingAudio(.pronunciationIncorrect):
+            return "Pronunciation Incorrect"
+        case .confusingAudio(.roadNameRepeated):
+            return "Road Name Repeated"
+        case .confusingAudio(.other):
+            return "Other"
+        case .routeQuality(.none):
+            return "Route Quality"
+        case .routeQuality(.routeNonDrivable):
+            return "Route Non Drivable"
+        case .routeQuality(.routeNotPreferred):
+            return "Route Not Preferred"
+        case .routeQuality(.alternativeRouteNotExpected):
+            return "Alternative Route Not Expected"
+        case .routeQuality(.routeIncludedMissingRoads):
+            return "Route Includes Missing Roads"
+        case .routeQuality(.routeHadRoadsTooNarrowToPass):
+            return "Route Had Roads Too Narrow To Pass"
```

Comment:
Should these items be in title case, or would sentence case be appropriate for labels that could be fairly long (especially when translated)? Either way, “to” should be lowercased in title case.

Reply:
sentence case works for me too. Switching to that.